.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!-- Thank you for taking the time to contribute to Odin-pantry. Please complete this entire template. -->
+
+## Because
+<!-- Summarize the purpose or reasons for this PR, e.g. what problem it solves or what benefit it provides. -->
+
+
+## This PR
```

Comment:
same suggestion as before; maybe something a little more obvious such as "What changes in this PR", etc

Reply:
Changed to: Changes in this PR